### ISSUE TITLE
NTR, pulmo: Neuroepithelial bodies in the alveolar ducts

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6391]
**Preferred term label:** Presence of neuroepithelial bodies in the alveolar ducts

**Synonyms**

**Definition (free text, please give PubMed ID)**
Presence of bombesin-immuno-positive neuroendocrine cell cluster (neuroepithelial bodies) in the alveolar ducts.

**Parent term (use hpo.jax.org/app)**
Interstitial pulmonary abnormality HP:0006530

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
NEHI ORPHA:217560

**Your nano-attribution (ORCID)**
LMU:chrapp, @kknoflach, @mgriese

**Comment by pnrobinson** [comment]
@mgriese can we define better what is the abnormality?
Pulmonary neuroepithelial bodies (NEB) in mammalian lungs are thought to function as airway O2 sensors that release serotonin (5-HT) in response to hypoxia.
So where are the NEB normally located? Why is the alveolar duct location abnormal?

**Comment by pnrobinson** [comment]
Label: Increased airway neuroepithelial bodies

**Comment by pnrobinson** [comment]
added
